core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/Schema.java

```diff
@@ -289,6 +301,12 @@ protected BaseBuilder(final Schema schema) {
             super(schema);
         }
 
+        /**
+         *
+         * @param id the id.
+         * @return this builder.
+         * @deprecated the ID should not be used. The ID should be supplied to the graph library separately
+         */
         public CHILD_CLASS id(final String id) {
```

Comment:
There are quite a few tests that still use the schema id. If you look at the call hierarchy of this id method then you will see them all. If the test is specifically testing the deprecated id field then it should be updated.